sklearn/ensemble/tests/test_forest.py

```diff
@@ -228,18 +231,20 @@ def check_importances(name, criterion, X, y):
         assert_less(np.abs(importances - importances_bis).mean(), 0.001)
```

Comment:
Ah okay. In addition to your proposed test, is it okay if I check the criterion and *iff* it's mae, change the tolerance to 0.01?